src/mem/port.hh

```diff
@@ -242,56 +245,63 @@ class MasterPort : public Port, public AtomicRequestProtocol,
     }
 };
 
+class M5_DEPRECATED MasterPort : public RequestPort
+{
+    public:
```

Comment:
two spaces :)

Reply:
Fixed! Let me know if that's not quite what you had in mind.